From 95bc1aa4f2901357d13170eaf941e59ded31459c Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Wed, 17 Sep 2008 10:49:04 -0500
Subject: [PATCH] ENGR00090522-2 mx51: set mmc caps in platform data

mx51: set mmc caps in platform data. The caps field is added
to specify the bus width.

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 arch/arm/mach-mx51/mx51_3stack.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx51/mx51_3stack.c b/arch/arm/mach-mx51/mx51_3stack.c
index 83c86ce..2a3a79b 100644
--- a/arch/arm/mach-mx51/mx51_3stack.c
+++ b/arch/arm/mach-mx51/mx51_3stack.c
@@ -338,6 +338,7 @@ EXPORT_SYMBOL(expio_intr_fec);
 #if defined(CONFIG_MMC_IMX_ESDHCI) || defined(CONFIG_MMC_IMX_ESDHCI_MODULE)
 static struct mxc_mmc_platform_data mmc_data = {
 	.ocr_mask = MMC_VDD_32_33,
+	.caps = MMC_CAP_4_BIT_DATA,
 	.min_clk = 400000,
 	.max_clk = 52000000,
 	.card_inserted_state = 1,
-- 
1.5.4.4

